javalib/src/main/scala/java/time/temporal/TemporalAmount.scala

```diff
@@ -0,0 +1,11 @@
+package java.time.temporal
+
+trait TemporalAmount {
+  def get(unit: TemporalUnit): Long
+
+  def getUnits: java.util.List[TemporalUnit]
```

Comment:
Methods in JDK classes should always have `()`: https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#parameterless-methods